### DISCUSSION TITLE
Submit your site power by Meghna Hugo Theme

### DISCUSSION THREAD

**Discussion by tfsomrat** [themefisher/meghna-hugo#174] [Show and tell]
Thanks for creating your site with Meghna-Hugo. Let's share your website to inspire others. You can share your website link here, we will add it to the list.

**Comment by Lyaaaaaaaaaaaaaaa** [comment]
Hi, thank you very much for this great template!
I edited it to suit my need.
[My website](https://www.lyaaaaagames.com/fr/)

**Comment by RAB1980** [comment]
Hi, thanks for this great theme!
My website: [www.romanbenvenuto.com](https://www.romanbenvenuto.com/)

**Reply by mehedi-sharif** [reply]
Hello @RAB1980 
Your website could not be reached.

**Reply by RAB1980** [reply]
> Hello @RAB1980 Your website could not be reached.

I access enter without errors.  Maybe it's a redirect issue.  https://www.romanbenvenuto.com 
can you try entering the link again?
Thanks!
Román

**Comment by ghost** [comment]
Hi,
have a look at a modified version of your beautiful theme at https://dk-hundetraining.de.
Yours, ukayy1

**Reply by mehedi-sharif** [reply]
your site has been added to our list. thank you

**Reply by ghost** [reply]
unfortunately I still can't find my site in your list, but thanks anyway ...

**Reply by mehedi-sharif** [reply]
https://github.com/themefisher/meghna-hugo/wiki/Popular-sites-powered-by-Meghna-Hugo here is all website links
